src/test/java/pl/byczazagroda/trackexpensesappbackend/controller/WalletControllerTest.java

```diff
@@ -436,10 +436,10 @@ void shouldReturnListOfAllWalletsByName() throws Exception {
         List<WalletDTO> listOfWalletsDTO = createListOfWalletsDTO();
         List<WalletDTO> foundedWalletsDTO = List.of(new WalletDTO(ID_OF_WALLET_2, NAME_OF_WALLET_2, CREATION_DATE_OF_WALLET_2));
         given(walletService.getAll()).willReturn(listOfWalletsDTO);
-        given(walletService.findByName(walletNameSearched)).willReturn(foundedWalletsDTO);
+        given(walletService.findAllByName(walletNameSearched)).willReturn(foundedWalletsDTO);
 
         // then
-        mockMvc.perform(get("/api/wallet/find/{name}", walletNameSearched)
+        mockMvc.perform(get("/api/wallet/list/{name}", walletNameSearched)
```

Comment:
We should retire "list" from our path. In my opinion the correct endpoint shoud be /api/wallet/{name}

Reply:
If we do it this way, we'll get an exception: Ambiguous handler methods mapped for "/api/wallet/{name}". This exception is related to the findById(java.lang.Long) and findByName(java.lang.String) methods.